Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/di/AppInjector.kt

```diff
@@ -9,4 +17,32 @@ object AppInjector {
         component = DaggerApplicationComponent.factory().create(app)
         component.inject(app)
     }
+
+    fun setup(activity: Activity) {
+        Timber.tag(TAG).d("Injecting %s", activity)
+
+        // Using lifecycle callbacks would be even more awesome,
+        // but Activity.onPreCreate isn't available for our minAPI
```

Comment:
should we branch out depending on API version?

Reply:
IMHO, no, as we would have different initialization behaviors depending on API version and could have bugs just on some  devices, increasing the chance for bugs that are not easily noticed/reproduced in tests. Too risky too little gain...